### ISSUE TITLE
Develop React Boilerplate Website

### ISSUE THREAD

**Issue by udaypydi** [udaypydi/react-simple-boilerplate#26]
Currently the boilerplate has a very naive website : http://react-simple-boilerplate.surge.sh/

Add all the components and proper documentation in the website with proper SEO.

**Comment by mundane140** [comment]
Do you need help with this one?

**Comment by udaypydi** [comment]
yeah , it would be great if someone takes this up.

**Comment by mundane140** [comment]
Okay, I can develop a react-app. For local state management I would prefer redux. 
Tell me more about the pages/sections you need on the web app.

**Comment by udaypydi** [comment]
A static app would be enough for this project. It would be great if we can render it server side (Maybe next.js).

I am aiming for something like this:
https://create-react-app.dev/docs/getting-started.

Currently we can add only the valid components to the project.

**Comment by mundane140** [comment]
Okay, I'll create a scaffold. Do you have the list of valid components ready? If that's ready, then I can start by adding empty components.

**Comment by mundane140** [comment]
I'll create a similar website and add a Getting started page.

**Comment by mundane140** [comment]
Do you want to keep the website code as a seperate repository? Or should I use a directory in the project for the website code?

**Comment by udaypydi** [comment]
For now you can add a `website` folder and get started with the project. however we have a plan to add a npm package for `react-simple-boilerplate` so that anyone can install and use
`react-simple-boilerplate create <ProjectName`. 
There is a plan for interactive cli as well.

**Comment by udaypydi** [comment]
I will add the list of tasks tonight. Meanwhile you can set up a project in `website` folder and get started. Post that you can add components to it.

**Comment by udaypydi** [comment]
> I'll create a similar website and add a Getting started page (content from readme.md).

**Comment by udaypydi** [comment]
@mundane140 I have added some basic stuff. It's not a monorepo yet but the projects work fine. I will refactor the folders and will be moving to monorepo using `yarn workspaces`. Meanwhile you can add the website components.
Here is the branch : `feat/boilerplate-website`

**Comment by udaypydi** [comment]
Need to fix the travis part , gitignore and a lot of things are pending. Unblocking your development.  Use the above branch to continue.

**Comment by akshay-nm** [comment]
This one too

**Comment by udaypydi** [comment]
If you can build the website, like I will share the content and components with you. 

I will share the tabs and pages content. You need to add them in the website. 
Can you help with that? 

Thanks!

**Comment by udaypydi** [comment]
@akshay-nm I am creating a new ticket for the components and the website edit. Closing this ticket as it seems to be done. Thanks!

**Comment by akshay-nm** [comment]
Okay @udaypydi, will check it out.
